app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -1060,4 +1061,85 @@ class LuciBuildService {
 
     return task.currentAttempt;
   }
+
+  /// Builder is defined in dart-internal:
+  /// https://dart-internal.googlesource.com/dart-internal/+/refs/heads/main/flutter-internal/flutter.star#33
+  static const _releaseBuilderName = 'Linux flutter_release_builder';
+
+  /// Reruns `Linux flutter_release_builder` for a release candidate [commit].
+  ///
+  /// Returns `false` if a rerun was not scheduled.
+  Future<bool> rerunDartInternalReleaseBuilder({
+    required CommitRef commit,
+    required int buildNumber,
+  }) async {
+    log.debug(
+      'rerunDartInternalReleaseBuilder(buildNumber=$buildNumber for $commit)',
+    );
+
+    // Because this is a large orchestrator build (a build that schedules many other sub-builds), and it is
+    // unlikely that every single build failed, we want to take advantage of the "retry_override_list" optional
+    // property, if able:
+    // https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/release/release_builder.py#162
+    final search = await _buildBucketClient.searchBuilds(
+      bbv2.SearchBuildsRequest(
+        predicate: bbv2.BuildPredicate(descendantOf: Int64(buildNumber)),
```

Comment:
convo: does this include all descendants or just the immediate children?

Reply:
Fixed.